e2e-tests/insurance/cypress/e2e/journeys/declarations/anti-bribery/anti-bribery.spec.js

```diff
@@ -28,6 +28,14 @@ context(
       cy.completeSignInAndGoToApplication({}).then(({ referenceNumber: refNumber }) => {
         referenceNumber = refNumber;
 
+        cy.completePrepareApplicationSinglePolicyType({ referenceNumber });
+
+        /**
+         * TODO: check this is in place in other declaration tests.
```

Comment:
ticvket number?

Reply:
Have added, created EMS-4096 to tackle this.